tracing-mock/src/subscriber.rs

```diff
@@ -31,15 +194,41 @@ pub fn mock() -> MockSubscriberBuilder {
     }
 }
 
+/// Create a [`MockSubscriberBuilder`] with a name already set.
+///
+/// This constructor is equivalent to `subscriber::mock().named(name)`.
+///
+/// For additional information and examples, see the [`subscriber`]
+/// module and [`MockSubscriberBuilder`] documentation.
+///
+/// [`subscriber`]: mod@crate::subscriber
 pub fn named(name: impl std::fmt::Display) -> MockSubscriberBuilder {
     mock().named(name)
 }
 
+/// A builder for the [`MockSubscriber`].
+///
+/// Use the methods on this struct to set expectations which are
+/// validated by the [`MockSubscriber`].
```

Comment:
in general we don't tend to write docs in the imperative form. how about
```suggestion
/// The methods on this builder set expectations which are then
/// validated by the constructed [`MockSubscriber`].
```